editorconfig.el

```diff
@@ -291,6 +291,8 @@ NOTE: Only the **buffer local** value of VARIABLE will be set."
 ;;;###autoload
 (add-hook 'find-file-hook 'edconf-find-file-hook)
 
+(add-to-list 'auto-mode-alist '("\\.editorconfig" . conf-unix-mode))
```

Comment:
How so? The [website](http://editorconfig.org/) states that the filename is `.editorconfig`, not something like `settings.editorconfig`.